docs/reference/announcements-release-notes/870/870-release-notes.md

```diff
@@ -457,6 +457,11 @@ The deployment experience is further simplified for Enterprise customers running
 - User tokens are used for deployments instead of machine-to-machine (M2M) tokens generated from a client ID and secret.
 - You no longer need to enter a client ID and secret in the deploy modal. Instead, simply choose a cluster (or stage for process applications) and deploy.
 
+:::note
+The simplified deployment experience is not supported when [Microsoft Entra ID is used as OIDC provider](../../../../self-managed/setup/guides/connect-to-an-oidc-provider/?authPlatform=microsoftEntraId#configuration).
```

Comment:
We could link to the announcement here, but optional.

Reply:
I added a link.